service_unavailable_page/web/public/internal/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Sorry, there’s a problem with the service - Register Trainee Teachers - GOV.UK</title>
```

Comment:
This should be 'Register trainee teachers'

Reply:
fixed